client/index_template.html

```diff
@@ -38,9 +38,11 @@
   <body>
     <script type="text/javascript">
       window.CELLXGENE = {};
+      backend_base_url = "{{BACKEND_BASE_URL}}";
+      prefix = backend_base_url ? backend_base_url : location.origin;
       window.CELLXGENE.API = {
-        prefix: `${location.origin}${location.pathname}api/`,
-        version: "v0.2/",
+          prefix: `${prefix}${location.pathname}api/`,
+          version: "v0.2/",
```

Comment:
Do we need to recompute any CSP hashes as a result of altering our script?

Reply:
No, I don't think so.

Reply:
You might want to try to test the CSP rules.  I wrote something up here on how to do that: https://github.com/chanzuckerberg/cellxgene/pull/1745

Reply:
The CSP header might have changed since then,  its probably best to grab it from a dev instance

Reply:
@seve,  I'm not familiar with how we handle hashes, but don't think we hash the index_template.html.  I see it for scripts and for main but not for this.

Reply:
Ooh thanks for pointing that out, @seve ! Yeah I have a feeling we'll need to update the CSP hash 👍 

@seve can you help us with that for this PR? Thank you!